LogMonitor/src/LogMonitor/Utility.h

```diff
@@ -44,4 +44,29 @@ class Utility final
         _In_ const std::wstring& From,
         _In_ const std::wstring& To
     );
+
+    static std::wstring STR_TO_W_STR(

```

Comment:
was suggesting that this doesn't go to the general `Utitliy.h`  but everything with telemetry is put in its own directory.